templates/details/libraries/libraries_layout.html

```diff
@@ -25,7 +25,7 @@
             <li class="p-side-navigation__item--title">
               <span class="p-side-navigation__text">
                 Publisher suggests
-                <div class="p-side-navigation__status u-hide--small">
+                <div class="p-side-navigation__status u-no-margin--left u-hide--small">
```

Comment:
could you add a check in here that doens't show this section `Publisher suggests` if there are no libraries?